packages/teams-js/src/public/pages.ts

```diff
@@ -374,14 +377,17 @@ export namespace pages {
       registerHandler('settings.remove', handleRemove, false);
     }
 
+    const setValidityStateLogger = configLogger.extend('setValidityState');
     /**
      * Sets the validity state for the configuration.
      * The initial value is false, so the user cannot save the configuration until this is called with true.
      * @param validityState - Indicates whether the save or remove button is enabled for the user.
      */
     export function setValidityState(validityState: boolean): void {
+      setValidityStateLogger('setting validity state to %i', validityState);
       ensureInitialized(FrameContexts.settings, FrameContexts.remove);
       if (!isSupported()) {
+        setValidityStateLogger('pages.config not supported, setValiditityState throwing errorNotSupportedOnPlatform');
```

Comment:
I understand you're adding it here because you're tracking a particular issue. Do we expect every single API to have this sort of logging around `ensureIinitialized` and `if(!isSupported())` at some point? If so, should we at least log an issue to track that? If not, should we log an issue to track removing this log statement once we're done with the investigations?

Reply:
Since you've tracked down the root cause, I'll remove logging from the specific functions in this file (but leave some of the more generic logging in place).

I do think it would be worth thinking of how to add logging for things like "isSupported" and "ensureInitialized" in a way that doesn't make the bundle double in size. I'll open a task to track that